### PR TITLE
Show post owener votes 

### LINKED ISSUE #133: Show to the owner user votes
When the user post will automatically see the percents of the votes

### PR DESCRIPTION
**Pull request by eslamheshamm**

### CODE REVIEW

**Review by alaasaedy — APPROVED** [review]
LGTM

File: src/components/ShareBtn/index.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { Popup } from 'reactjs-popup';
...
-  const [text, setText] = useState('Copy post url');
+  const [text, setText] = useState('Share');
 
   const clipboard = useClipboard({
     copiedTimeout: 600
   });
   const handleClick = React.useCallback(() => {
     clipboard.copy(url); // programmatically copying a value
-    setText('coped');
+    setText('Shared');
```
Comment (MAlngaawy) [inline src/components/ShareBtn/index.js:14]: why you change this to, _share_ and _shared_?
the user doesn't share anything!
he just copied the post link

File: src/components/PostSection/index.jsx [hunk 2]
```diff
@@ -182,7 +187,16 @@ const PostSection = ({
...
+const PostImage = ({
+  images,
+  img,
+  voted,
+  handleVote,
+  _id,
+  totalVotes,
+  ownedByCurrentUser
+}) => {
+  console.log(img);
```
Comment (MAlngaawy) [inline src/components/PostSection/index.jsx]: remove console.log

**Review by MAlngaawy — CHANGES_REQUESTED** [review]

File: src/components/ShareBtn/index.js (hunk 1 above)
Comment (eslamheshamm) [reply src/components/ShareBtn/index.js:14]: when he copy the link he sharing it so when it copied we can call it shared

File: src/components/PostSection/index.jsx (hunk 2 above)
Comment (eslamheshamm) [reply src/components/PostSection/index.jsx]: okay

**Review by MAlngaawy — APPROVED** [review]

**Pull request merged**